vector/src/main/java/fr/gouv/tchap/activity/TchapLoginActivity.java

```diff
@@ -641,9 +598,6 @@ public void afterTextChanged(Editable editable) {
                 RegistrationManager.getInstance().attemptRegistration(this, this);
                 onWaitingEmailValidation();
             }
-        } else if (mMode == MODE_ACCOUNT_CREATION) {
-            // Update the tchap platform if an email is available
```

Comment:
I would just let the if, with an empty body, to not go in the else case. It's maybe ok

Reply:
Actually we want to go in the else case here even if mMode == MODE_ACCOUNT_CREATION